ping-thing-client.mjs

```diff
@@ -256,7 +265,7 @@ async function pingThing() {
           {
             headers: {
               "Content-Type": "application/json",
-              "Token": VA_API_KEY,
+              Token: VA_API_KEY,
```

Comment:
this looks strange?

Reply:
JavaScript takes care of it regardless of the quotes. Quotes are needed if the key contains special characters like hyphens(-).

Pushed a commit to add quotes to `Token`